lib/js_defer.js

```diff
@@ -1456,6 +1462,29 @@ deferJsNs.DeferJs.prototype.noDeferCreateElementOverride = function() {
   };
 };
 
+deferJsNs.DeferJs.prototype.deferInsertScriptTag = function() {
```

Comment:
Is it always safe to defer an `insertBefore`? This will alter the excecution order of the inserted scripts.